config/config-mysql_apache_jvm.yaml

```diff
@@ -0,0 +1,83 @@
+  prometheus_exec/mysql:
+    exec: ./prometheus_exporters/mysqld_exporter --web.listen-address=":{{port}}" --no-collect.global_variables  # this last flag reduces the amount of generated metrics, can be removed to enable all (600+) MySQL metrics
+    port: 9104
+    env:
+    - name: DATA_SOURCE_NAME  # DATA_SOURCE_NAME and its value are needed to connect to the MySQL server, alternatively you could use a .my.cnf configuration file
+      value: username:password@(host:port)/db_name  # replace all value here with your MySQL server credentials
+
+  # https://github.com/Lusitaniae/apache_exporter
+  prometheus_exec/apache:
+    exec: ./prometheus_exporters/apache_exporter --telemetry.address=":{{port}}"
+    port: 9117
+
+  # https://github.com/prometheus/jmx_exporter
+  prometheus_exec/jvm:
+    exec: java -jar ./prometheus_exporters/jmx_exporter.jar {{port}} exporter_configs/jvm_config.yaml # see jvm_config.yaml for JMX configuration, notably the "hostPort" key
+    port: 9404
+
+processors:
+  filter:
+    metrics:
+      include:
+        match_type: regexp
+        metric_names: # Allowing all JVM and Apache metrics, but whitelisting MySQL ones to filter hundreds of unused ones
+                - mysql_global_status_commands_total
+                - mysql_global_status_threads_running
+                - mysql_global_status_handlers_total
+                - apache_.*
+                - jmx_.*
+                - java_.*
+                - promhttp_.*
+
+  metricstransform/mysql:
+    transforms:
+      - metric_name: mysql_global_status_commands_total
+        action: update
+        new_name: mysql/command_count
+        operations:
+          # change data type from double -> int64
+          - action: toggle_scalar_data_type
+      - metric_name: mysql_global_status_threads_running
+        action: update
+        new_name: mysql/thread_count
+      - metric_name: mysql_global_status_handlers_total
+        action: update
+        new_name: mysql/handler_count
+
+  metricstransform/java:
+    transforms:
+      - metric_name: java_lang_GarbageCollector_CollectionCount
+        action: update
+        new_name: jvm/gc/count
+
+  metricstransform/apache:
+    transforms:
+      - metric_name: promhttp_metric_handler_requests_total
+        action: update
+        new_name: apache/request_count
+        operations:
+          - action: aggregate_labels
+            label_set: []
+            aggregation_type: sum
+            # change data type from double -> int64
+          - action: toggle_scalar_data_type
+
+exporters:
+  stackdriver:  
+    project:  # stackdriver will use the default GCE credentials
+    #metric_prefix: agent.googleapis.com/
+
+service:
+  pipelines:
+    metrics:
+      receivers: [prometheus_exec/mysql, prometheus_exec/apache, prometheus_exec/jvm]  # Remember to remove unwanted receivers from this list
+      processors: [filter, metricstransform/mysql, metricstransform/java]
```

Comment:
Missing `apache`?

Reply:
Also consider making this three diff metrics pipelines, i.e.

```yaml
metrics/mysql:
      receivers: [prometheus_exec/mysql]
      processors: [filter/mysql, metricstransform/mysql]
      exporters: [stackdriver]

metrics/apache:
      receivers: [prometheus_exec/apache]
      processors: [metricstransform/apache]
      exporters: [stackdriver]

...
```

That will be slightly more efficient

Reply:
If it is more efficient I will do this, but as of now it's meant for a user to be able to just remove a receiver from the list to modify the config to their needs, and having them in separate pipelines is a worst experience! If the performance gain is minimal I think this may be better for the config modification experience for the user!

Reply:
Why would that be a worse experience? Instead of deleting a receiver they could just delete a pipeline?

I feel like it would make the config file better modularised / readable which should be a better user experience.

Reply:
Oh I was thinking they'd ignore un-needed receivers by simply removing them from the same list of `receivers` at the bottom, but I'll modify it for modularity!

Reply:
To me removing a pipeline is just as straightforward as removing the receiver.

Seems like you only added the `mysql` pipeline atm?